### PR TITLE
AR : Fix missing part & some translate in site.json

### PR DESCRIPTION
**Pull request by 3imed-jaberi**
Some details :
1. I've added a part that is in the English version and is missing here ; 
```
    "dependencies": {
      "link": "docs/meta/topics/dependencies",
      "text": "Dependencies"
    }
  <!-- from `Dependencies` to `تبعيات` -->
```
2. Fix translation of meetups ; from `لقاءات` to `مُلْتَقَيات` .. 
3.  ~~Fix translation of docs ; from `التوثيق` to `المستندات` ..~~

### CODE REVIEW

**Comment by 3imed-jaberi** [comment]
Hi @Aissaoui-Ahmed , 
Please check these changes and thank you ❤️ ..
If you can give me your email address ✌️

**Comment by Aissaoui-Ahmed** [comment]
I appreciate your interest I've left a comment on [React ](https://github.com/reactjs/ar.reactjs.org/issues/122#issuecomment-511190921) about words
in the file site.json don't update it before complete translation 
you can contact me on [Facebook](https://www.facebook.com/a.aissaoui)

**Comment by 3imed-jaberi** [comment]
I'm sorry my bro ( @Aissaoui-Ahmed ) ..
At REACT we have the methodology of work and the glossary that we build ..
For example, when the difference in the translation of a word we must discuss ..
I've read your comments there and fixed the TAB bug ..

File: locale/ar/site.json [hunk 1]
```diff
@@ -87,7 +87,7 @@
   },
   "docs": {
     "link": "docs",
-    "text": "التوثيق",
+    "text": "المستندات",
```
Comment (Aissaoui-Ahmed) [inline locale/ar/site.json]: المستندات = documents
التوثيق = documentation

**Review by Aissaoui-Ahmed — CHANGES_REQUESTED** [review]

File: locale/ar/site.json (hunk 1 above)
Comment (3imed-jaberi) [reply locale/ar/site.json]: وثائق = documents ... !

Comment (3imed-jaberi) [reply locale/ar/site.json]: الوثائق  = documents ... !

**Review by Aissaoui-Ahmed — APPROVED** [review]

**Pull request merged**